cosmoz-page-router.js

```diff
@@ -121,6 +135,7 @@
 		}
 
 		ready() {
+			super.ready();
```

Comment:
@dotpointer This here is the reason why `cosmoz-page-router` wasn't even initialized as an element.

Reply:
Ok, thanks for the information.